<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="mahendrakodi-lambdatest-nl8omvlwbxc" failures="1" tests="3" name="Lambda Test Assessment" time="65.711" errors="0" timestamp="30 Jun 2023 20:32:25 GMT">
  <testcase classname="com.lambda.test.Tests" name="scenario1" time="2.046"/>
  <testcase classname="com.lambda.test.Tests" name="scenario2" time="8.986"/>
  <testcase classname="com.lambda.test.Tests" name="scenario3" time="4.795">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Please fill in the fields] but found [Please fill out this field.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [Please fill in the fields] but found [Please fill out this field.]
	at com.lambda.test.Tests.scenario3(Tests.java:90)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
... Removed 8 stack frames]]>
    </failure>
  </testcase> <!-- scenario3 -->
</testsuite> <!-- Lambda Test Assessment -->
